Makefile

```diff
@@ -37,7 +37,7 @@ css:
 js:
 	mkdir -p $(BUILD)
 	bash static/js/vendor.jsh > $(BUILD)/vendor-v2.js
-	bash static/js/all.jsh > $(BUILD)/all.js
+	npm run build-assets:webpack
```

Comment:
we may want to be careful here as these make instructions get called on production -- we'll need to make sure webpack is setup on prod before merging. I can own this

Reply:
thank you for the information